test/unit/lodash.spec.js

```diff
@@ -0,0 +1,709 @@
+            expect(elem2 instanceof Object).to.eql(true);
+            expect(params).to.have.length(2);
+        });
+        
+        it('should first object has right properties', function(){
+            expect(params[0]).to.have.property('data');
+            expect(params[0].data[0].owner).to.eql('Adam');
+            expect(params[0].data[1].owner).to.eql('Tom');
+            expect(params[0].data[2].owner).to.eql('John');
+        });
+        
+        it('should second object has right properties', function(){
+            expect(params[1]).to.have.property('data');
+            expect(params[1].data[0].cats).to.eql(3);
+            expect(params[1].data[1].dogs).to.eql(5);
+            expect(params[1].data[2].ostrich).to.eql(1);
+        });
+        
+        it('should return new object fold from 2 objects', function(){
+            expect(_.merge.apply(_, datasets.merge(params))).to.have.property('data');
+            expect(_.merge.apply(_, datasets.merge(params)).data[0].owner).to.eql('Adam');
+            expect(_.merge.apply(_, datasets.merge(params)).data[0].cats).to.eql(3);
+            expect(_.merge.apply(_, datasets.merge(params)).data[1].owner).to.eql('Tom');
+            expect(_.merge.apply(_, datasets.merge(params)).data[1].dogs).to.eql(5);
+            expect(_.merge.apply(_, datasets.merge(params)).data[2].owner).to.eql('John');
+            expect(_.merge.apply(_, datasets.merge(params)).data[2].ostrich).to.eql(1);
+        });
+    });
+
+
+    describe('omit', function () {
+        describe('omit1', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.omit1();
+            });
+
+            it('should check types', function(){
+                var elem2 = params[1];
+                expect(typeof elem2).to.eql('string');
+                expect(params).to.have.length(2);
+            });
+            it('should return ommited object', function(){
+                var obj = {};
+
+                _.times(50, function(index){
+                    obj['number' + index] = _.random(20);
+                });
+                expect(_.omit.apply(_, datasets.omit1(obj))).to.eql(_.omit(obj, params[1]));
+            });
+        });
+
+        describe('omit2', function () {
+            var params;
+            beforeEach(function(){
+                params = datasets.omit2();
+            });
+
+            it('should check types', function(){
+                var elem = params[1];
+                expect(elem instanceof Array).to.eql(true);
+                expect(params).to.have.length(2);
+            });
+            it('should return ommited object', function(){
+                var obj = {};
+
+                _.times(50, function(index){
+                    obj['number' + index] = _.random(20);
+                });
+                expect(_.omit.apply(_, datasets.omit2(obj))).to.eql(_.omit(obj, params[1]));
```

Comment:
You are supposed to check if user ommited correct elements instead of trying to just comparing the same values from different context.